src/components/CaptureFilter.js

```diff
@@ -306,23 +220,23 @@ function Filter(props) {
               <TextField
                 htmlFor="grower-id"
                 id="grower-id"
-                label="Grower ID"
+                label="Grower Account ID"
                 placeholder="e.g. 2, 7"
                 value={growerId}
```

Comment:
Can we change the names of these variables to match the new microservice schema?

Reply:
Yes, I think we should, but didn't want to do it until we are fully switched over.  Maybe that can be a cleanup issue after we confirm everything is working?  Unless you think it's a good idea to do now.

Reply:
I'm happy to do this under a cleanup issue.